A few hiccups with tutorial #4

I've experienced a few hiccups going through the tutorial https://github.com/containers/buildah/blob/master/docs/tutorials/04-include-in-your-build-tool.md 

Adding Buildah as a Go dependency "as-is" fails with errors:

```
# github.com/containers/storage/drivers/btrfs
../go/pkg/mod/github.com/containers/storage@v1.24.5/drivers/btrfs/btrfs.go:8:10: fatal error: btrfs/ioctl.h: No such file or directory
 #include <btrfs/ioctl.h>
          ^~~~~~~~~~~~~~~
compilation terminated.
# pkg-config --cflags  -- devmapper
Package devmapper was not found in the pkg-config search path.
Perhaps you should add the directory containing `devmapper.pc'
to the PKG_CONFIG_PATH environment variable
No package 'devmapper' found
pkg-config: exit status 1
# github.com/mtrmac/gpgme
../go/pkg/mod/github.com/mtrmac/gpgme@v0.1.2/data.go:4:11: fatal error: gpgme.h: No such file or directory
 // #include <gpgme.h>
           ^~~~~~~~~
compilation terminated.
```
This is fixed by installing the following packages:
```
zypper in libbtrfs-devel libgpgme-devel device-mapper-devel
```
I believe, mentioning the required packages/libraries would make the tutorial more welcoming for new users.

Some context: I'm using a VitualBox VM with
```
OS: openSUSE Leap 15.2
go version go1.15.6 linux/amd64
```

--- a/docs/tutorials/04-include-in-your-build-tool.md
+++ b/docs/tutorials/04-include-in-your-build-tool.md
@@ -13,14 +13,28 @@ In this tutorial I'll show you how to create a simple CLI tool that creates an i
 
 Bootstrap the installation of development dependencies of Buildah by following the [Building from scratch](https://github.com/slinkydeveloper/buildah/blob/master/install.md#building-from-scratch) instructions and in particular creating a directory for the Buildah project by completing the instructions in the [Installation from GitHub](https://github.com/containers/buildah/blob/master/install.md#installation-from-github) section of that page.
 
-Now let's bootstrap our project. Assuming you are in the directory of the project, run:
+Now let's bootstrap our project. Assuming you are in the directory of the project, run the following to initialize the go modules:
 
 
 ```shell
 go mod init
 ```
 
-To initialize the go modules. Now import Buildah as a dependency:
+Next, we should import Buildah as a dependency. However, make sure that you have the following
+developer packages installed:
+
+```shell
+dnf install btrfs-progs-devel gpgme-devel device-mapper-devel
+```
+
+Depending on your Linux distribution, the names of the packages can be slightly different. For instance, on
+OpenSUSE it would be
+
+```shell
+zypper in libbtrfs-devel libgpgme-devel device-mapper-devel
+```
+
+Now import Buildah as a dependency:
 
 ```shell
 go get github.com/containers/buildah

